tests/forte/processors/data_augment/algorithms/UDA_test.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for Unsupervised Data Augmentation.
+"""
+import tempfile
+import unittest
+import torch
+import texar.torch as tx
+from forte.processors.data_augment.algorithms.UDA import UDAIterator
+from texar.torch.utils.shapes import get_rank
+
+
+class UDAPipelineTest(unittest.TestCase):
```

Comment:
Let's not name this "pipeline" though. This would be confused with Forte pipeline.